src/utils.ts

```diff
@@ -21,3 +21,10 @@ export const remove0x = (str: string): string => {
     return str
   }
 }
+
+export const formatNumber = (num: any): number => {
```

Comment:
Should this be `num: string`?

Is there a function which does this in `core-utils`? If not I think it's fine to include here but eventually probably a good idea to move it there

Reply:
i copied this from ethers formatter, which i think is supposed to keep the input general since it may begin as a number too

https://github.com/ethers-io/ethers.js/blob/6c43e20e7a68f3f5a141c74527ec63d9fe8458be/packages/providers/src.ts/formatter.ts#L165-L170

i'll move it into core-utils in a followup PR :)